### ISSUE TITLE
New `ITensors.op` overloading syntax

### ISSUE THREAD

**Issue by mtfishman** [christopherdavidwhite/potts-gs-magic#1]
Hey Christopher,

I just noticed your paper, looks nice (and great title!).

Perhaps you already know, but there is an updated interface for creating custom site types and overloading the `op` function, described in a post by Miles [here](http://www.itensor.org/docs.cgi?vers=julia&page=formulas/sitetype_basic). Pointing it out in case you want to update your Potts definitions (which may be useful to have in ITensors as well).

-Matt

**Comment by christopherdavidwhite** [comment]
Nope, hadn't seen that---thanks for pointing it out. If I end up spending much more time with this code, I'll update it and send you a pull request.
